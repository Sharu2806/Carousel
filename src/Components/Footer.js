import React from 'react';

const Footer = (props) => {
  return (
    <footer className="footer">
      <button tabIndex="0" onClick={props.onClickPrev} className="prev">Prev</button>
      <button tabIndex="0" onClick={props.onClickNext} className="next">Next</button>
    </footer>
  );
}

export default Footer;
