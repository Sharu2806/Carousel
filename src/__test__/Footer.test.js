import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../Components/Footer';

test('Match Screenshot', () => {
  const Component = renderer.create(<Footer />);
  expect(Component.toJSON()).toMatchSnapshot();
});
