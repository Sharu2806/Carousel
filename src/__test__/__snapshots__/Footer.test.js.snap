// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match Screenshot 1`] = `
<footer
  className="footer"
>
  <button
    className="prev"
    tabIndex="0"
  >
    Prev
  </button>
  <button
    className="next"
    tabIndex="0"
  >
    Next
  </button>
</footer>
`;
